src/scss/components/file/mixins/hxFileControl.scss

```diff
@@ -0,0 +1,66 @@
+@import "vars";
+
+@mixin hxFileControl(
+  $pseudo-state: null,
+  $btnState: null,
+  $disabledState: null
+) {
+  @if $pseudo-state == focus {
+    box-shadow: $focus-glow;
+  } @else if $pseudo-state == active {
+    ~ label > hx-file-input {
+      // TODO refactor button mixins
+      @if $btnState == btn {
+        &.hxBtn {
+          background-color: $cyan-700;
+          border-color: transparent;
+          color: $gray-0;
+        }
+      } @else if $btnState == hxPrimary {
+        // TODO refactor button mixins
+        &.hxBtn.hxPrimary {
+          background-color: $cyan-700;
+          border-color: transparent;
+          color: $gray-0;
+        }
+      } @else if $btnState == hxTertiary {
+        // TODO refactor button mixins
+        &.hxBtn.hxTertiary {
+          background-color: transparent;
+          border-color: transparent;
+          color: $cyan-700;
+        }
+      }
+    }
```

Comment:
* Please leave these styles as they are currently defined in [`src/scss/components/file/_hx-file-control.scss`](https://github.com/HelixDesignSystem/helix-ui/blob/master/src/scss/components/file/_hx-file-control.scss#L34-L53) and remove this entire block of styles.

-----
🚨 Button mixins need to be updated before we can do anything to replace these styles. 

The only state we have the capability to style at the moment is for 

```css
hx-file-control > input[type="file"].mock-focus ~ label > hx-file-input
```
which should apply the same styles as

```css
hx-file-control > input[type="file"]:focus ~ label > hx-file-input
```

_NOTE_: Element selectors (i.e., `label` and `hx-file-input`) should rarely ever appear in a mixin.  If a mixin should include selectors in its definition, it should only include pseudo-classes (e.g., `:invalid`, `:hover`, etc.) or pseudo-elements (e.g., `::before`, `::after`, etc.), nothing else.